CONTRIBUTING.md

```diff
@@ -366,3 +366,79 @@ duration = float_or_none(video.get('durationMs'), scale=1000)
 view_count = int_or_none(video.get('views'))
 ```
 
+### Inline values
+
+ Always inline values.
```

Comment:
This is basically incorrect. Extracting variables is fine for eliminating code duplication and improving readability for complex expressions. It's not fine to extract variables used only once and move them to the opposite part of extractor file making linear reading flow problematic.

Reply:
Okay, thanks. I reworded it based on what you wrote.